client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -222,15 +222,20 @@ function PurchaseMetaOwner( { owner } ) {
 
 function PurchaseMetaPrice( { purchase } ) {
 	const translate = useTranslate();
-	const { priceText, productSlug } = purchase;
+	const { productSlug, productDisplayPrice } = purchase;
```

Comment:
It's minor since none of these changed files are using TS, but can we add `productDisplayPrice` to the TypeScript type for this data, which I believe is in the following location? 

https://github.com/Automattic/wp-calypso/blob/c58bb2ba436e0576c9c11936ade83e95c40471f5/client/lib/purchases/types.ts#L16

(I based this on the fact that `purchase` is selected into `PurchaseMeta` [here](https://github.com/Automattic/wp-calypso/blob/c58bb2ba436e0576c9c11936ade83e95c40471f5/client/me/purchases/manage-purchase/purchase-meta.jsx#L62) and that calls [this function](https://github.com/Automattic/wp-calypso/blob/c58bb2ba436e0576c9c11936ade83e95c40471f5/client/state/purchases/selectors/get-by-purchase-id.js#L12) which does specify a type.)

Reply:
All fixed